docs/src/InsolationExamples.md

```diff
@@ -43,19 +46,24 @@ plot_day_lat_insolation(days, lats, F0, "YlOrRd", title, "insol_example1.png")
 
 ### Insolation with smaller obliquity
 ```@example
-using CLIMAParameters # hide
-using CLIMAParameters.Planet # hide
-struct EarthParameterSet <: AbstractEarthParameterSet end # hide
-const param_set = EarthParameterSet() # hide
+import CLIMAParameters as CP
+import Insolation
+import Insolation.Parameters as IP
+
+FT = Float64
+include(joinpath(pkgdir(Insolation), "parameters", "create_parameters.jl"))
+param_set = create_insolation_parameters(FT)
+
 include("plot_insolation.jl") # hide
-γ0 = obliq_epoch(param_set) # hide
-ϖ0 = lon_perihelion_epoch(param_set) # hide
-e0 = eccentricity_epoch(param_set) # hide
+γ0 = IP.obliq_epoch(param_set) # hide
+ϖ0 = IP.lon_perihelion_epoch(param_set) # hide
+e0 = IP.eccentricity_epoch(param_set) # hide
 days, lats, F0 = calc_day_lat_insolation(365, 180, param_set) # hide
 
 # decrease γ to 20.0°
-CLIMAParameters.Planet.obliq_epoch(::EarthParameterSet) = deg2rad(20.0)
-γ1 = obliq_epoch(param_set)
+# TODO: verify this
```

Comment:
The plots in the docs look fine. Is that what you mean by verify here Charlie?

Reply:
Yeah, I think I just wanted to make sure the plots looked okay.